src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable no-nested-ternary */
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+    <p className="Person__partner">
+      {person.isMarried
+        ? person.sex === 'f'
+          ? `${person.partnerName} is my husband`
+          : `${person.partnerName} is my wife`
```

Comment:
Try to avoid nested ternary operators because it's reduce code readable. For example, you can create a variable and assign ternary statement to it. As for me, not a big mistake, so if you want you can just ignore it))

[ESLint](https://eslint.org/docs/latest/rules/no-nested-ternary#resources)